src/components/AddItemModal.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import DefaultModal, { Overlay, ButtonField } from './DefaultModal';
+import Logo from '../molecules/Logo';
+import TextInput from './TextInput';
+import CancelButton from './CancelButton';
+import MainButton from './MainActionButton';
+
+const Body = styled.div`
+	height: 20em;
+`;
+
+const Desc = styled.p`
+	color: black;
+	font-size: 14px;
+`;
+
+const BodyLogo = styled.div`
+	grid-area: logo;
+	justify-content: center;
+	align-items: center;
+`;
+
+const InputField = styled.div`
+	grid-area: input;
+	margin: 0 2em;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-rows: 1fr 1.5fr;
+	grid-template-areas:
+		'name date'
+		'weight grade'
+	color: black;
+	justify-content: center;
+	align-items: center;
+`;
+
+const InputName = styled(TextInput)`
+	grid-area: name;
+`;
+
+const InputDate = styled(TextInput)`
+	grid-area: date;
+`;
+
+const InputWeight = styled(TextInput)`
+	grid-area: weight;
+`;
+
+const InputGrade = styled(TextInput)`
+	grid-area: grade;
+`;
+
+const Cancel = styled(CancelButton)`
+	grid-area: left;
+`;
+
+const Submit = styled(MainButton)`
+	grid-area: right;
+`;
+
+export interface AddItemInterface {
+	onSubmit: (data: AddItemData) => void;
+}
+
+export interface AddItemData {
+	name: string;
+	date: string;
+	weight: string;
+	grade: string;
+}
+
+function AddItemModal({ onSubmit }: AddItemInterface) {
+	const [isOpen, setStatus] = useState('');
```

Comment:
Two things here; could this be a boolean? and lets stick to the React State naming convention. 

Here's what I think it could be.
```typescript
const [open, setOpen] = useState(false);
```

Reply:
Sounds good! I set it to a string so I could directly set `className` to `closed` through the `open` variable; would it be more efficient to conditionally set the `className`?

Reply:
Also, why should `false` be the initial state?